frontend/src/__tests__/cypress/cypress/support/commands/application.ts

```diff
@@ -237,6 +257,10 @@ Cypress.Commands.overwriteQuery('findAllByTestId', function findAllByTestId(...a
   return enhancedFindByTestId(this, ...args);
 });
 
+Cypress.Commands.add('step', (description: string) => {
+  cy.log(`\n--- 🔷 Step: ${description} 🔷---\n`);
+});
+
```

Comment:
What about printing Steps with a counter like:

```
let stepCounter = 0;

Cypress.Commands.add('step', (description) => {
  stepCounter += 1;
  const stepMessage = `Step ${stepCounter.toString().padStart(2, '0')}: ${description}`;
  cy.log(stepMessage);
  console.log(stepMessage);
});
```

Reply:
Let me give this a try, there is a plugin we can use for this. I'm not sure how the steps would work when running multiple tests which might be a problem.

Reply:
If we go with the above the 'before' method needs have the counter added for each test which i feel is unnecessary overhead. There is a plugin I referenced in the Framework document that can be used, would you like me to implement that?

Reply:
Yes, and an existing plugin for printing steps is preferable, but I think it's better to add it as a new PR

Reply:
Agreed, thanks for confirming.